diff --git a/CMakeLists.txt b/CMakeLists.txt
index 39a5650..4e13e6c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -9,8 +9,6 @@ else()
   option (werror "Treat warnings as errors." TRUE)
 endif()
 
-enable_testing ()
-option (enable_lcov "Generate Coverage Reports" ON)
 option (enable_mirclient "Enable overlay using mirclient" ON)
 
 set(CMAKE_MODULE_PATH "${CMAKE_SOURCE_DIR}/cmake" "${CMAKE_MODULE_PATH}")
@@ -68,9 +66,6 @@ include_directories(${GIO_UNIX2_INCLUDE_DIRS})
 pkg_check_modules(JSONGLIB REQUIRED json-glib-1.0)
 include_directories(${JSONGLIB_INCLUDE_DIRS})
 
-pkg_check_modules(DBUSTEST REQUIRED dbustest-1>=14.04.0)
-include_directories(${DBUSTEST_INCLUDE_DIRS})
-
 pkg_check_modules(SQLITE REQUIRED sqlite3)
 include_directories(${SQLITE_INCLUDE_DIRS})
 
@@ -111,14 +106,3 @@ add_subdirectory(liblomiri-url-dispatcher)
 add_subdirectory(tools)
 add_subdirectory(gui)
 add_subdirectory(po)
-add_subdirectory(tests)
-
-# coverage reporting
-find_package(CoverageReport)
-get_property(COVERAGE_TARGETS GLOBAL PROPERTY COVERAGE_TARGETS)
-get_property(COVERAGE_TESTS GLOBAL PROPERTY COVERAGE_TESTS)
-ENABLE_COVERAGE_REPORT(
-  TARGETS ${COVERAGE_TARGETS}
-  TESTS ${COVERAGE_TESTS}
-  FILTER /usr/include ${CMAKE_BINARY_DIR}/*
-)
